lib/jwt_signed_request/verify.rb

```diff
@@ -50,8 +50,12 @@ def secret_key
       @secret_key ||= stored_key.fetch(:key) { raise MissingKeyIdError }
     end
 
+    JWT_BEARER_REGEX = /^Bearer ([^*]+)$/
```

Comment:
This won't match if `Bearer` is missing. Is that intentional? You might want to swap it to include both.

```rb
\A(Bearer\s)?([^*]+)\z
```

If you don't need to match on the `Bearer` word, I'd suggest swapping `^` for `\A` and `$` for `\z` to perform end of string matches instead of end of line.

Reply:
updated in: 016c50f
🙇  @jacobbednarz